webpack.config.js

```diff
@@ -22,151 +24,108 @@ const SCOPE_VERSION = JSON.stringify(versionHash)
 
 console.info('This build version hash is', versionHash, '\n')
 
-const config = {
-	mode: buildMode,
-	devtool: isDev ? false : 'source-map',
-	devServer: {
-		historyApiFallback: true,
-		noInfo: true,
-		overlay: true,
-	},
-	entry: {
-		ncvuecomponents: path.join(__dirname, 'src', 'index.js'),
-		install: path.join(__dirname, 'src', 'install.js'),
-		...glob.sync('src/components/*/index.js').reduce((acc, item) => {
-			const name = item
-				.replace('/index.js', '')
-				.replace('src/components/', 'Components/')
-			acc[name] = path.join(__dirname, item)
-			return acc
-		}, {}),
-		...glob.sync('src/directives/*/index.js').reduce((acc, item) => {
-			const name = item
-				.replace('/index.js', '')
-				.replace('src/directives/', 'Directives/')
-			acc[name] = path.join(__dirname, item)
-			return acc
-		}, {}),
-		...glob.sync('src/functions/*/index.js').reduce((acc, item) => {
-			const name = item
-				.replace('/index.js', '')
-				.replace('src/functions/', 'Functions/')
-			acc[name] = path.join(__dirname, item)
-			return acc
-		}, {}),
-		...glob.sync('src/mixins/*/index.js').reduce((acc, item) => {
-			const name = item
-				.replace('/index.js', '')
-				.replace('src/mixins/', 'Mixins/')
-			acc[name] = path.join(__dirname, item)
-			return acc
-		}, {}),
-	},
-	output: {
-		path: path.resolve(__dirname, './dist'),
-		publicPath: '/dist/',
-		filename: '[name].js',
-		library: {
-			type: libraryTarget,
-			name: ['NextcloudVue', '[name]'],
-		},
-		umdNamedDefine: true,
+webpackConfig.entry = {
+	index: path.join(__dirname, 'src', 'index.js'),
```

Comment:
@skjnldsv Comparing https://github.com/nextcloud/nextcloud-vue/blob/master/webpack.config.js with https://github.com/nextcloud/nextcloud-vue/blob/v7.3.0/webpack.config.js I think this also needs to go into the umd section here:
https://github.com/nextcloud/nextcloud-vue/blob/e5999882e2ae4fe22b2c72e3671ef509e1e8a36e/webpack.config.js#L159-L161